Application/Frontend/lib/mongodb.tsx

```diff
@@ -1,7 +1,27 @@
 const { MongoClient, ServerApiVersion } = require("mongodb");
 import { getMongoDbUri } from "@/lib/dbConfig";
 
-// Create a MongoClient with a MongoClientOptions object to set the Stable API version
+/**
```

Comment:
Good comment and good implementation of the Mongodb to one folder to remove confusion and align better with our framework so far.